packages/now-ruby/test/fixtures/09-activesupport/Gemfile.lock

```diff
@@ -0,0 +1,26 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
+    activesupport (6.0.2.1)
+      concurrent-ruby (~> 1.0, >= 1.0.2)
+      i18n (>= 0.7, < 2)
+      minitest (~> 5.1)
+      tzinfo (~> 1.1)
+      zeitwerk (~> 2.2)
+    concurrent-ruby (1.1.6)
+    i18n (1.8.2)
+      concurrent-ruby (~> 1.0)
+    minitest (5.14.0)
+    thread_safe (0.3.6)
+    tzinfo (1.2.6)
+      thread_safe (~> 0.1)
+    zeitwerk (2.2.2)
+
+PLATFORMS
+  ruby
+
+DEPENDENCIES
+  activesupport (~> 6.0)
+
+BUNDLED WITH
+   1.17.3
```

Comment:
Does it make a difference which bundle version is used? Can you upgrade to 2.x.x? Most other tests are using `2.0.1`.

Reply:
I've installed ruby `v2.5.7` which is used in production by ZEIT and this is the out-of-the-box `bundler` version. But no problem will bundle with the latest `bundler` version 👍

Reply:
Interesting, I think I'm confusing the static ruby builds with the serverless functions. The static ruby is 2.6 right now and the latest bundler.